src/queries/tokenIdOpengraphQuery.ts

```diff
@@ -11,29 +11,144 @@ export const tokenIdOpengraphQuery = `query TokenIdOpengraphQuery($tokenId: DBID
         name
         description
          media {
-                  ... on ImageMedia {
-                    __typename
-                    previewURLs {
-                      small
-                      medium
-                      large
-                    }
-                    fallbackMedia {
-                      mediaURL
-                    }
-                  }
-                  ... on VideoMedia {
-                    __typename
-                    previewURLs {
-                      small
-                      medium
-                      large
-                    }
-                    fallbackMedia {
-                      mediaURL
-                    } 
-                  }
-         }
+          ... on AudioMedia {
+            __typename
+            previewURLs {
+              small
+              medium
+              large
+            }
```

Comment:
nit: to minimize copy+paste typos in future migrations, this massive media block of `... on AudioMedia`, etc. i think can be a shared exported string that we import and inline

e.g.
```
// mediaQuerySubstring.ts
export const mediaQuerySubstring = `
  ... on AudioMedia {
     etc
  }
  etc
`

// tokenIdOpengraphQuery.ts (and other files)
import { mediaQuerySubstring } from '...'

export const tokenIdOpengraphQuery = `
   ...
   definition {
      name
      description
      media {
         ${mediaQuerySubstring}
      }
  ....
`

Reply:
great idea!